job.go

```diff
@@ -0,0 +1,100 @@
+package flux
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+const (
+	// DefaultQueue is the queue to use if none is set.
+	DefaultQueue = "default"
+
+	// ReleaseJob is the method for a release job
+	ReleaseJob = "release"
+
+	// PriorityBackground is priority for background jobs
+	PriorityBackground = 100
+
+	// PriorityInteractive is priority for interactive jobs
+	PriorityInteractive = 200
+)
+
+var (
+	ErrNoSuchJob        = errors.New("no such release job found")
+	ErrNoJobAvailable   = errors.New("no release job available")
+	ErrUnknownJobMethod = errors.New("unknown job method")
+)
+
+type JobStore interface {
```

Comment:
Ah. they're mentioned in the `release/releaser.go` (which `jobs/worker.go` depends on). :(